src/wx/guiinit.cpp

```diff
@@ -3330,10 +3330,36 @@ bool MainFrame::BindControls()
         sc->SetValidator(wxUIntValidator(&o)); \
     } while (0)
         {
+#ifndef NO_ONLINEUPDATES
             // Online Auto Update check frequency
             getrbi("UpdateNever", gopts.onlineupdates, 0);
             getrbi("UpdateDaily", gopts.onlineupdates, 1);
             getrbi("UpdateWeekly", gopts.onlineupdates, 7);
+#else
+            wxWindowList &children = d->GetChildren();
+            std::vector<wxWindow*> forDeletion;
+            for (wxWindowList::Node *node = children.GetFirst(); node; node = node->GetNext())
```

Comment:
This is cool, but is this really necessary? Not displaying the controls would probably be enough. Fine with me either way though.

Reply:
Is there a way to hide all elements from a `<object class="sizeritem">` or `<object class="wxBoxSizer">`? I did not find a easy way to do this, like I am doing at the moment (by selecting something with label/name equals to what I want to remove).

Since I was already doing the entire selection, I decided to delete these controls.